test/contracts/consumer/fabric8auth/verify_interactions.go

```diff
@@ -0,0 +1,51 @@
+package fabric8auth
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/pact-foundation/pact-go/dsl"
+)
+
+// SimpleGetInteraction executes a simple GET request against Pact Mock server
+// to verify the interaction
+func SimpleGetInteraction(pact *dsl.Pact, endpoint string) func() error {
+	var test = func() error {
+		u := fmt.Sprintf("http://localhost:%d%s", pact.Server.Port, endpoint)
+		req, err := http.NewRequest("GET", u, nil)
+
+		if err != nil {
```

Comment:
Please have a look at the other test files. I bet that this file is only supposed to be used in testing, right? Then name the file `verify_interactions_test.go` and put the code into `package fabric8auth_test`; otherwise it will be included in the main binary which we don't want. It is a good idea to also pass in the test handle to this function fail the test right here using `require.NoError(t, err)`.

Reply:
> Please have a look at the other test files. I bet that this file is only supposed to be used in testing, right?

Well, actually everything under `test/contracts/` package is supposed to be used only in testing and not to be included in the main binary.

Reply:
Correct. Everything should be in a `..._test` package.

Reply:
I re-arranged the packages and files in https://github.com/fabric8-services/fabric8-wit/pull/2393/commits/39d2da7bfc809e44fd2a827ba15c099b0d231d30

Reply:
You don't have to move everything in its own directory. Test and main code can live next to each other in one directory. Just make sure you have `package maincode` for production code and `package maincode_test` for good blackbox test code:

```
maincode/
|- foo.go
|- foo_test.go
```

Reply:
@kwk Thanks, rearranged it again.